exopy_hqc_legacy/instruments/drivers/visa/agilent_pna.py

```diff
@@ -80,14 +80,26 @@ def read_formatted_data(self, meas_name=''):
             self.selected_measure = meas_name
         else:
             meas_name = self.selected_measure
+            
+        # get the endianess of the output so that the data is read properly
+        data_endianess = self._pna.data_endianess
+        if data_endianess == 'little':
+            is_big_endian = False
+        elif data_endianess == 'big':
+            is_big_endian = True
+        else:
+            raise InstrIOError(cleandoc('''Agilent PNA did not define 
+                                        endianess properly'''))
```

Comment:
Same as below the property ensure a valid answer.
```suggestion
is_big_endian = self._pna.data_endianess == "big"
```